compiler/src/dotty/tools/dotc/core/tasty/Attributes.scala

```diff
@@ -1,6 +1,23 @@
 package dotty.tools.dotc.core.tasty
 
+import dotty.tools.tasty.TastyFormat
+
 class Attributes(
-  val scala2StandardLibrary: Boolean,
-  val explicitNulls: Boolean,
-)
+  val booleanTags: List[Int],
```

Comment:
couldn't this be a bitset?

Reply:
It could. I will try to optimize this in a separate PR. Will merge as is to unblock the other PRs.